backend/Main.java

```diff
@@ -15,7 +15,7 @@ public static void main (String[] args)
     //and set the available paths from the "paths.txt" file. Then this cities instances are returned in a citylist.
     
     System.out.println ("Here is the list of cities."); 
-    input.print (",");
+    System.out.println(input.send());
```

Comment:
The input will not be printed directly. But rather it will now be sent to a stream to be displayed. While coupling, just change the system. out to bufferstream's out.